pkg/generated/filterer/civilgovernment.go

```diff
@@ -134,70 +134,10 @@ func (f *GovernmentContractFilterers) StartGovernmentContractFilterers(client bi
 	wg.Add(1)
 	go func() {
 		filterFunc := func() {
-			startBlock := f.eventToStartBlock["NewConstProposal"]
-			e, pevents := f.startFilterNewConstProposal(startBlock, []*model.Event{})
+			startBlock := f.eventToStartBlock["NewConstSet"]
```

Comment:
Hm.. so there was also a go-common update as part of this commit

Reply:
I'm wondering why this was overwritten. Did the event name change or is there a new event?

Reply:
yes, there was a go-common update since I pulled in updates to go-common for the error reporting.  There was also a recent update to update the contract wrappers and add the abi/bin files directly to the go-common repo, so this seems like it was a result of that.  Any issues here?  It's possible it was a name change, but would have to dig in a little to see.

Reply:
We should look into that because if there were name changes, we would have mismatch of events names in our DB, which we would need to account for in the processor logic

Reply:
looking into it now

Reply:
Looks like Nick updated the government contract on Jan 22 which removes `NewConstProposal` and adds `NewConstSet`. I can make the updates in the processor to handle the new event, although the context looks different.

https://github.com/joincivil/Civil/commit/539cb6238d86a29adcdef68a589fbb2e1b46e40f#diff-bcae085adca39b4527e2a84e2598ad1b

Reply:
Ohh ok, I still have to write that logic to process government events so no changes need to be made, i just mentioned it for the logic going forward. But yeah good thing we updated that now!